src/main/registry.ts

```diff
@@ -316,7 +316,7 @@ export class Registry implements IRegistry {
         if (platform === 'win32') {
             path_env = `${envPath};${envPath}\\Library\\mingw-w64\\bin;${envPath}\\Library\\usr\\bin;${envPath}\\Library\\bin;${envPath}\\Scripts;${envPath}\\bin;${process.env['PATH']}`;
         } else {
-            path_env = `${envPath}:${process.env['PATH']}`;
+            path_env = `${envPath}:${envPath}/bin:${process.env['PATH']}`;
```

Comment:
if possible, avoid doing _any_ `PATH` hacking, if it can possibly be avoided.

Reply:
the path here is not used when activating the environment anymore.